string droneName
string groupName
---
string result
